examples/dynamoathenamigration/migration.go

```diff
@@ -410,6 +411,10 @@ func (t *task) fromS3ToChan(ctx context.Context, dataObj dataObjectInfo, eventsC
 			return false, trace.Wrap(err)
 		}
 
+		if ev.GetID() == "" {
```

Comment:
Are we good with non-UUID ids or with the all-zero UUID?

Reply:
reworked it in https://github.com/gravitational/teleport/pull/29193/commits/8ca8c403bf2853dabdb6716dbfe5ce6db06d533c